### DISCUSSION TITLE
Missing features

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84634] [Mobile]
...

**Comment by imanmahjoubi** [comment]
@BlueLockArc Thank you for your feedback. Could you provide further details about the specific functionalities that were missing?

**Reply by verhovsky** [reply]
Deleting Issues
<img width="334" alt="Screenshot 2024-01-07 at 01 26 24" src="https://github.com/community/community/assets/5687998/38a847f7-8054-438c-9fd4-c696204041f2">

**Reply by verhovsky** [reply]
Programming language statistics for a repo

<img width="332" alt="Screenshot 2024-01-07 at 01 26 48" src="https://github.com/community/community/assets/5687998/b61f197b-958c-4142-875e-aeec5d2848d0">
